### PR TITLE
Add homebrewed wine package

### PR DESCRIPTION
**Pull request by KevinMartin**
This take 30+ minutes to install on the machine. Bad choke point.

### CODE REVIEW

**Comment by viktorbenei** [comment]
Looks good! 👍

**Comment by viktorbenei** [comment]
Just tested, and I don't think `brew install wine` is enough. I got:

```
+ brew install wine
wine: XQuartz is required to install this formula.
You can install with Homebrew Cask:
  brew install Caskroom/cask/xquartz
You can download from:
  https://xquartz.macosforge.org
Error: An unsatisfied requirement failed this build.
```

**Comment by KevinMartin** [comment]
@viktorbenei thanks for that. Completely forgot about that one.

**Comment by KevinMartin** [comment]
@viktorbenei would it be possible to make the builds public so we can see what the errors are?

**Comment by viktorbenei** [comment]
Not yet (you can vote on the feature request at: https://bitrise.uservoice.com/forums/235233-general/suggestions/8795035-allow-public-or-publicly-viewable-apps), but the error I copy pasted was not from the PR checks, it was from a simple script step with the content `brew install wine`.

**Comment by KevinMartin** [comment]
@viktorbenei Would you be able to give me some insight then as too why the 4 checks are failing now that I added `xquartz`?

**Comment by viktorbenei** [comment]
@KevinMartin the PR check are not yet properly implemented, as noted in the related section of the readme https://github.com/bitrise-io/osx-box-bootstrap#request-a-tool-to-be-pre-installed

Right now the PR checks are just guides for us ;)

**Comment by KevinMartin** [comment]
Excellent then. Didn't noticed the italicized part.

Looking forward to any comments on the PR.

**Comment by viktorbenei** [comment]
Looks good - we'll try to include this in the next OS X VMs. We won't add it to the existing VM configurations, but it'll be in the upcoming Xcode 7.3 beta VM, and every other VM after that.

Just one question: right now you only report the version of `wine` - shouldn't you include Quartz's or something else's version too? These things usually help to debug issues, when your build breaks on a new VM and it turns out that the new VM now uses a newer version of a tool which is no longer compatible with a previous version. I'm not that familiar with `wine`, so it's completely up to you, just wanted to mention.

**Comment by KevinMartin** [comment]
I'm not familiar with Xquartz much. I've never run it directly and I can't seem to find a way to get the version number from the command line.

**Comment by viktorbenei** [comment]
No problem at all, just wanted to check if there's any other version number which would be useful to have in the system reports :)

**Comment by viktorbenei** [comment]
New Xcode 7.3 beta (3) is available now - we'll start to prepare the new VM soon!

**Comment by KevinMartin** [comment]
Excellent! :+1:

**Comment by viktorbenei** [comment]
Sorry, we had to delay the preparation a bit, we'll do it today.

**Comment by viktorbenei** [comment]
Btw the issue was that we downloaded the new Beta right away when it was available on https://developer.apple.com/xcode/download/ , but it seems the download link was broken on the website, and it was pointing to an older beta version, not to beta 3.

**Comment by KevinMartin** [comment]
No worries. I'll be waiting 😜

File: playbook.yml [hunk 1]
```diff
@@ -141,7 +141,12 @@
     - name: run brew installer script
       script: installers/install_brew.sh creates=/usr/local/bin/brew
     - homebrew: update_homebrew=yes upgrade_all=no
+    # Tap some brews
+    - homebrew_taps:
```
Comment (viktorbenei) [inline playbook.yml]: actually you don't need to tap `caskroom/cask` anymore, so we'll remove this

**Comment by viktorbenei** [comment]
The new VM is now available - thanks again for the PR! @KevinMartin - http://blog.bitrise.io/2016/02/10/xcode-7-3-beta-3-is-now-available.html

**Comment by viktorbenei** [comment]
@KevinMartin - do you use `wine` on bitrise.io?

There's an issue with it right now, because http://www.freedesktop.org/ is down (`brew install wine` depends on a package hosted on freedesktop.org), and this means that we can't finish the preparation of the new Xcode 7.3 beta image.

We'll wait a bit, but if the issue is not solved soon (it seems the site is down for at least 5 hours now) we'll have to remove `wine` from the setup, at least for this VM version (Xcode 7.3 beta 4).

**Comment by viktorbenei** [comment]
We'll continue the setup without `wine` - if you'd need the previous VM which included `wine` just let us know and we'll make it available again.

**Comment by KevinMartin** [comment]
@viktorbenei, Yes. I do use `wine` on bitrise.io :/

**Comment by KevinMartin** [comment]
Thanks for letting me know, btw.

**Comment by viktorbenei** [comment]
@KevinMartin we'll try to do a new VM setup as it seems that https://www.freedesktop.org/ is finally available again; if it succeeds we'll update the Xcode 7.3 beta VMs, if it would fail we'll expose the previous `beta 3` VM as a temporary solution.

**Comment by viktorbenei** [comment]
If everything works, and `wine` can be installed, it'll be available in about an hour. I guess we'll have to come up with a plan for cases like this, when a single dependency can't be installed for an extended period of time..

**Comment by viktorbenei** [comment]
Good news @KevinMartin - `wine` install went through successfully, the new VM should be available in about 20 mins!

**Comment by viktorbenei** [comment]
VM passed all of our tests, and should be available now. We updated the blog post & the bitrise-io/bitrise.io system reports with the info, `wine` is pre-installed again on the Xcode 7.3 beta stack.

Sorry for the inconvenience.

**Comment by KevinMartin** [comment]
No worries. Thanks for the ping and info!

**Pull request merged**